modules/nodes-nodes-audit-log-advanced.adoc

```diff
@@ -0,0 +1,137 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-audit-log.adoc
+
+[id="nodes-nodes-audit-log-advanced-{context}"]
+= Advanced Audit
+
+The advanced audit feature provides several improvements over the
+basic audit functionality, including fine-grained events filtering and multiple output back ends.
+
+To enable the advanced audit feature, provide the following values in the
+`openshift_master_audit_config` parameter:
+
+----
+openshift_master_audit_config={"enabled": true, "auditFilePath": "/var/lib/origin/oscp-audit.log", "maximumFileRetentionDays": 14, "maximumFileSizeMegabytes": 500, "maximumRetainedFiles": 5, "policyFile": "/etc/origin/master/adv-audit.yaml", "logFormat":"json"}
```

Comment:
Where and on what object do you set this.

Reply:
@sferich888 I believe it was the _/etc/origin/master/adv-audit.yaml_. Is his file in 4.x?
If so, how do users edit files on OpenShift systems? Previously, the user could SSH in; do they need physical access to edit such files?